lib/sles4sap_publiccloud.pm

```diff
@@ -115,7 +115,7 @@ sub run_cmd {
     delete($args{timeout});
     delete($args{runas});
 
-    $self->{my_instance}->wait_for_ssh(timeout => $timeout);
+    $self->{my_instance}->wait_for_ssh(timeout => $timeout, scan_ssh_host_key => 1);
```

Comment:
Is the idea to put `scan_ssh_host_key => 1` not to perturbate SAP tests? To let SAP test works exactly like before?

Could you explain what we should check to decide if we can omit  `scan_ssh_host_key` here too ?